src/hazelcore/semantics/CursorInfo.re

```diff
@@ -1091,11 +1093,33 @@ module Exp = {
           ty,
         )
       }
-    | CaseZA(NotInHole, _, rules, zann) =>
-      Typ.cursor_info(~steps=steps @ [1 + List.length(rules)], ctx, zann)
     | ApPaletteZ(NotInHole, _, _, _) =>
       syn_cursor_info_zoperand(~steps, ctx, zoperand)
     }
+  and syn_cursor_info_rule =
```

Comment:
maybe out of scope for this PR, but it would be nice for the cursor inspector to show the glb of all the other branches as the expected type instead of just "any type" in synthetic position

Reply:
What would it show in the case where the other branches still had type inconsistencies? Would it just show "any type" in that case?

Reply:
branches only have type inconsistencies on the whole branch expression in analytic position. in any case, you just treat those branches as ?...

Reply:
![image](https://user-images.githubusercontent.com/21255175/81837683-637f9400-9502-11ea-808e-4317e9f40fd2.png)
Are you talking about how the cursor inspector shows "any type" if the cursor is where the red circle is?

Reply:
https://github.com/hazelgrove/hazel/pull/215/commits/cb5392f6579b509f3a8798801991e3595aae5271